### PR TITLE
Update Turkish Docs

### PR DESCRIPTION
**Pull request by DursunCanPoyraz**
### Turkish (Türkçe) docs support

**Work In Progress**

### Done:

- [x] Introduction
- [x] Setting Up
- [x] Management

### Things to do:

- [ ] Extend

### CODE REVIEW

**Comment by DursunCanPoyraz** [comment]
I'll translate the extend folder later. I will get help from my friend because code knowledge is required. My friend is busy now.
Publish the last 5 commit on the site? I will take a look at the error.

**Comment by luceos** [comment]
Thank you for your hard work 😍 

Please understand we don't blindly merge translations as any offensive language directly impacts our reputation. My preference is having at least one reviewer with Turkish native proficiency. Hope you understand that this delays the PR.

File: docs/.vuepress/config/locales/tr/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    lang: 'tr',
+    title: 'Flarum Dokümantasyon',
```
Comment (tolgaaaltas) [inline docs/.vuepress/config/locales/tr/index.js:3]: "Doküman" Fransızca bir kelimedir. Onun yerine Türkçe olan "Belge" kelimesini kullanmayı tercih edebilirsin. 
[Source](https://sozluk.gov.tr/)

File: docs/tr/contributing.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+Flarum, [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md) kodlama standardını ve [PSR- 4](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-4-autoloader.md) otomatik yükleme standardı. Bunun da ötesinde, [diğer stil kurallarına](https://github.com/flarum/core/blob/master/.styleci.yml) uyarız. Mümkün olduğunda PHP 7 tür ipucu ve dönüş türü bildirimlerini ve satır içi belgeler sağlamak için [PHPDoc](https://docs.phpdoc.org/) kullanıyoruz. Katkılarınızda kod tabanının geri kalanı tarafından kullanılan stili deneyin ve taklit edin.
+
+* Ad alanları tekil olmalıdır (ör. `Flarum\Discussion`, not `Flarum\Discussions`)
+* Arayüzlerin sonuna `Interface` eklenmelidir (ör. `MailableInterface`)
+* Abstract sınıflarının önüne `Abstract` yazılmalıdır (ör `AbstractModel`)
+* Özelliklerin sonuna `Trait` eklenmelidir (ör. `ScopeVisibilityTrait`)
+
+### JavaScript
+
+Flarum'un JavaScript'i çoğunlukla [Airbnb Stil Kılavuzu](https://github.com/airbnb/javascript)'nu takip eder. Satır içi dokümantasyon sağlamak için [ESDoc](https://esdoc.org/manual/tags.html) kullanıyoruz.
```
Comment (tolgaaaltas) [inline docs/tr/contributing.md:106]: "Doküman" yerine "belge" diyebiliz.

File: docs/tr/extensions.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+## Uzantıları Bulmak
+
+Flarum, çoğu açık kaynaklı ve ücretsiz olan geniş bir uzantı ekosistemine sahiptir. Yeni ve harika uzantılar bulmak için, Flarum'un topluluk forumundaki [Extensions](https://discuss.flarum.org/t/extensions) etiketini ziyaret edin. Resmi olmayan [Extiverse](https://extiverse.com/) de harika bir kaynaktır.
+
+## Uzantıları Yükleme
+
+Flarum gibi, uzantılar da SSH kullanılarak [Composer](https://getcomposer.org) aracılığıyla yüklenir. Tipik bir uzantı yüklemek için:
+
+1. `composer.json` dosyasını içeren klasöre `cd` komutuyla gidin.
+2. Besteciyi çalıştırmak için `composer require COMPOSER_PACKAGE_NAME` gereklidir. Bu, uzantının dokümantasyonu tarafından sağlanmalıdır.
```
Comment (tolgaaaltas) [inline docs/tr/extensions.md:18]: "Doküman" yerine "belge" diyebiliz.

File: docs/tr/themes.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- Karanlık modu ve renkli üstbilgi değiştir
+- Bir logo ve favicon yükleyin (tarayıcı sekmelerinde gösterilen simge)
+- Özel üstbilgiler ve altbilgiler için HTML ekleyin
+- Öğelerin görüntülenme şeklini değiştirmek için [özel LESS/CSS](#css-theming) ekleyin
+
+## CSS Tema Oluşturma
+
+CSS, tarayıcılara bir web sayfasının öğelerinin nasıl görüntüleneceğini söyleyen bir stil sayfası dilidir. Renklerden yazı tiplerine, eleman boyutuna ve konumlandırmadan animasyonlara kadar her şeyi değiştirmemizi sağlar. Özel CSS eklemek, Flarum kurulumunuzu bir temayla eşleşecek şekilde değiştirmenin harika bir yolu olabilir.
+
+Bir CSS eğitimi bu dokümantasyonun kapsamı dışındadır, ancak CSS'nin temellerini öğrenmek için birçok harika çevrimiçi kaynak vardır.
```
Comment (tolgaaaltas) [inline docs/tr/themes.md:19]: "Doküman" yerine "belge" diyebiliriz.

File: docs/.vuepress/config/locales/tr/index.js (hunk 1 above)
Comment (DursunCanPoyraz) [reply docs/.vuepress/config/locales/tr/index.js]: Thanks, fixed it.

- https://github.com/flarum/docs/pull/154/commits/bb0c788c46394d8acac0a45c1172ffc627dfd33e

File: docs/tr/contributing.md (hunk 2 above)
Comment (DursunCanPoyraz) [reply docs/tr/contributing.md]: Thanks, fixed it.

- https://github.com/flarum/docs/pull/154/commits/bb0c788c46394d8acac0a45c1172ffc627dfd33e

File: docs/tr/extensions.md (hunk 3 above)
Comment (DursunCanPoyraz) [reply docs/tr/extensions.md]: Thanks, fixed it.

- https://github.com/flarum/docs/pull/154/commits/bb0c788c46394d8acac0a45c1172ffc627dfd33e

File: docs/tr/themes.md (hunk 4 above)
Comment (DursunCanPoyraz) [reply docs/tr/themes.md]: Thanks, fixed it.

- https://github.com/flarum/docs/pull/154/commits/bb0c788c46394d8acac0a45c1172ffc627dfd33e

**Review by halilhanbadem — APPROVED** [review]
No problem has been detected.

**Comment by tolgaaaltas** [comment]
I checked quickly and 99.9% okay. We can improve with @DursunCanPoyraz later if anything needs. I'm working on my thesis right now.

**Review by askvortsov1 — CHANGES_REQUESTED** [review]
Thank you very much for this! I don't really have much to say about the translations since I don't speak Turkish, but I'm not 100% sure about leaving the `extend` docs in English. It might be simpler to leave them as "This page is under construction" until they are implemented, like was done with the Chinese translations of those pages?

**Comment by DursunCanPoyraz** [comment]
@askvortsov1 I cannot translate pages about coding into Turkish. If the pages made so far are approved, I will get help for the remaining pages. I have 2 friends. They will help.

**Comment by tankerkiller125** [comment]
@DursunCanPoyraz What @askvortsov1 is saying is that instead of those pages being in English (as they are now) all of the extend pages should be changed to say "Coming soon" in Turkish.

**Comment by DursunCanPoyraz** [comment]
Coming soon article has been added for pages not translated into Turkish. All documents will be translated into Turkish as soon as possible.

**Review by askvortsov1 — APPROVED** [review]
Thank you very much!

**Review by tankerkiller125 — APPROVED** [review]

**Comment by justoverclockl** [comment]
anyone can help me to upload the italian version? how can i start this on github?

**Comment by Littlegolden** [comment]
> anyone can help me to upload the italian version? how can i start this on github?

Hava you translated the basic documents? Is everything working properly in the local environment?

**Pull request merged**